### DISCUSSION TITLE
Need jgroups 5.1.8 final compatible with java 8

### DISCUSSION THREAD

**Discussion by Sagar-Singari** [belaban/JGroups#541] [Q&A] [answered]
Hi @belaban,

I'm trying to use jgroups 5.1.8 final with java 8 and I'm getting "class file has wrong version 55.0, should be 52.0" error.
Is there a way I can use this version without migrating to java 11?

Thanks.

**Comment by belaban** [comment, marked as answer]
Hi @bobsag 
5.x has Java 11 as baseline, so running it under Java 8 won't work.

**Reply by Sagar-Singari** [reply]
Thanks @belaban!
